source/elements/oneMKL/source/domains/dft/descriptor.rst

```diff
@@ -154,6 +191,55 @@ of all necessary parameters.
 **Descriptor class member table:** :ref:`onemkl_dft_descriptor_member_table`
 
 
+.. _onemkl_dft_descriptor_assignment_operator:
+
+Descriptor class assignment operators
++++++++++++++++++++++++++++++++++++++
+
+The assignment operators allow assignment operations.
```

Comment:
```suggestion
```
I think its fine without this.

Reply:
Done in [9f90533](https://github.com/oneapi-src/oneAPI-spec/pull/490/commits/9f9053386a24e589f26254ba79f7157a693803aa).